db/migrate/20230523085326_create_search_stats.rb

```diff
@@ -0,0 +1,18 @@
+class CreateSearchStats < ActiveRecord::Migration[7.0]
+  def change
+    create_table :search_stats do |t|
+      t.string :keyword, null: false
+      t.integer :ad_count, null: false, default: 0
+      t.integer :link_count, null: false, default: 0
+      t.bigint :total_result_count, null: false, default: 0
+      t.text :raw_response
+      t.bigint :user_id, null: false
```

Comment:
Should it be a [references](https://edgeguides.rubyonrails.org/active_record_migrations.html#references) instead? Not only store the `user_id`, but we also need an index on it. 😄 

```suggestion
      t.references :user, null: false, foreign_key: true
```

Reply:
Fixed in 6318643c91b3493f8752730f8fb84771e234a8a1